client/app/components/Input/InputRadioGroup.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React, { useState } from 'react';
+import css from './InputRadioGroup.scss';
+import type { Option } from './utils';
+
+export type Props = {
+  name?: string,
+  options: Option[],
+  value?: any,
+}
+
+export function InputRadioGroup({
+  name,
+  options,
+  value: propValue,
+}: Props) {
+  return (
+    <div className={css.wrapper}>
```

Comment:
This should be replaced with a `<fieldset>`.

Reply:
Sounds good! 👍 I have added `<fieldset>` around this component. It did introduce some unwanted CSS styling that I disabled but it also prevents `display: flex` and `flex-direction: row` from being applied to it. That is why I had to keep the wrapper div to allow for `flex-direction: row`. Otherwise, items will be displayed in a column. Let me know your thoughts on this

Reply:
Hmm okay, I did some more research and I think it should suffice to put `role="radiogroup"` on this div.

Reply:
Cool, I have made this change! Confused why the circleci build is not happening 🤔 I have squashed my commits. I also synced with the upstream master as explained in the wiki. I have rebased my branch with the master so it is on the top. None of this seems to reset the circleci build. I did some Google and it might be caused by something enabled under the repo's "settings" tab ?